src/openproduct/producten/models/eigenaar.py

```diff
@@ -49,25 +49,26 @@ class Eigenaar(BaseModel):
         help_text=_("Een korte unieke aanduiding van een vestiging."),
     )
 
-    klantnummer = models.CharField(
-        _("Klantnummer"),
-        help_text=_("generiek veld voor de identificatie van een klant."),
-        max_length=50,
+    partijnummer = models.CharField(
```

Comment:
This is a breaking change, might be good to check with @alextreme  how we deal with this, versioning wise

Reply:
I would recommend to keep this field as klantnummer and update the help text to mention that this is to identify a klant or a partij in Open Klant or a klant in any other CRM system. This last aspect is also important as we don't make use of the Klantinteractie API from Open Product, it can also be any other registry of klantinformation

In my eyes the confusion lies with Open Klant that there isn't a Klant anymore in OpenKlant :)

We won't change this within the scope of this PR. If this doesn't work for municipalities using OpenKlant or other systems we'll expand upon it or rename it, but I don't see the need for this breaking change atm.